pilot/pkg/security/authz/model/model.go

```diff
@@ -165,52 +163,6 @@ func (m *Model) MigrateTrustDomain(tdBundle trustdomain.Bundle) {
 	}
 }
 
-// Update rules for ambient based on listener name. Because the Waypoint Proxy receives
-// the destination as envoy://listener_name/, Envoy will not have a port to evaluate. This
-// will precalculate the RBAC rules with regard to destination port for any matches and replace
-// as necessary. For not matches, we add an `any: true` as it will always match. For value match,
-// we replace with a nil, which the generator replaces with an `any: true` if there are no other
-// rules to evaluate. We leave non-matching ports so they do not match any requests.
-func (m *Model) AmbientDestinationPortAdaptations(listenerName string) {
```

Comment:
Since this is being removed... are the waypoint proxies able to handle destination port policies directly now? I didn't see an integration test to test this

Reply:
Yes, it's based on the port in the authority (service port or pod port).